How to setup agent hooks (#340)
buildkite/agent-stack-k8s: README.md

Our JSON schema can also be used with editors that support JSON Schema by configuring your editor to validate against the schema found [here](./cmd/linter/schema.json).

## How to setup agent hooks

> This section explains how to setup agent hooks when running Agent Stack K8s. In order for the agent hooks to work, as we all know we need them to be present on the instances where the agent runs.

Reviewer: "as we all know" - _do_ we all know?

```suggestion
This section explains how to setup agent hooks when running Agent Stack K8s. In order for the agent hooks to work, they must be present on the instances where the agent runs.
```
